src/Plugin/migrate/process/VideoImageSource.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
 namespace Drupal\custom_peertube_migration\Plugin\migrate\process;
+if (!defined('PCDM_URI')) {
```

Comment:
The [`define()` method](https://www.php.net/manual/en/function.define.php) operates globally.  It would be better to use a `const` declaration to define this constant within the current namespace.